### PR TITLE
Update README.md #25646

### PR DESCRIPTION
**Pull request by philiplee13**
Description - Update README.md #25646

Added anchor tags for "contributing-to-docs" and "localization-readmes"

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/language en

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/be93b718-a6df-402a-b4a4-855ba186c97d/deploy-status)](https://app.netlify.com/sites/kubernetes-io-master-staging/deploys) [![GitHub release](https://img.shields.io/github/release/kubernetes/website.svg)](https://github.com/kubernetes/website/releases/latest)
 
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
+<br>[Contributing to the docs](#contributing-to-the-docs)
+<br>[Localization ReadMes](#localization-readmemds)
```
Comment (onlydole) [inline README.md:7]:
```suggestion

[Contributing to the docs](#contributing-to-the-docs)

[Localization ReadMes](#localization-readmemds)
```

I'd recommend using newlines and returns as opposed to the `<br>` tag

**Review by onlydole — CHANGES_REQUESTED** [review]
Looks good to me with one nit related to Markdown vs. HTML

File: README.md (hunk 1 above)
Comment (philiplee13) [reply README.md]: Thanks @onlydole! Still getting my started with open source so appreciate the suggested changes. I've updated the PR, please let me know if I can do anything else

**Comment by onlydole** [comment]
Thank you, @philiplee13! 

/lgtm
/approve

**Pull request merged**